apps/cli/src/models.ts

```diff
@@ -43,7 +42,7 @@ class BaseTransfer<
 
   $data?: DTO;
 
-  $fromDTO: (id: ID, json: JsonObject) => DTO = () => {
+  $fromDTO: (id: ID, json: Record<string, any>) => DTO = () => {
```

Comment:
@fwkoch I ran into the same issue you were with the jsonObject stuff and typescript. I think it is fixed here.